xunitparser.py

```diff
@@ -167,7 +186,12 @@ def parse_root(self, root):
     def parse_testsuite(self, root, ts):
         assert root.tag == 'testsuite'
         ts.name = root.attrib.get('name')
+        ts.raw_timestamp = root.attrib.get('timestamp')
+        ts.timestamp = to_datetime(root.attrib.get('timestamp'))
         ts.package = root.attrib.get('package')
+        ts.hostname = root.attrib.get('hostname')
+        ts.time = to_timedelta(root.attrib.get('time'))
```

Comment:
Could you update the parsing of 'time' to take into account that JUnit files sometimes contain human-readable human-readable strings rather than xsd:decimal?

Depending on what definition of junit.xsd you take as gospel,  either JUnit or xunitparser is broken. There is, strangely, no universally agreed upon canonical xunit.xsd definition, despite the format's widespread use.

JUnit tests will generate a localized human readable string in the time attribute, e.g. "1.234,56" with Norwegian or "1,234.56" in US locale. xunitparser balks on this because xunitparser takes takes as read that the time attribute is    an xsd:decimal type and not a human readable string.